bin/hdfcoinc/pycbc_page_snrratehist

```diff
@@ -26,17 +28,44 @@ parser.add_argument('--output-file')
 parser.add_argument('--bin-size', type=float, default=0.1)
 parser.add_argument('--x-min', type=float, default=8.0)
 parser.add_argument('--trials-factor', type=int, default=1)
-parser.add_argument('--cumulative', action='store_true')
+parser.add_argument('--h-iter-inc-background', type=int, default=None,
+                    help='Indicate which inclusive background to plot '
+                         'with the foreground triggers if there were '
+                         'any hierarchical removals done. Choosing 0 '
+                         'indicates plotting the inclusive background prior '
+                         'to any hierarchical removals. Choosing 1 indicates '
+                         'the inclusive background after the loudest '
+                         'foreground trigger was removed. Choosing another '
+                         'integer gives the inclusive background after that '
+                         'integer number of hierarchical removals. Choosing '
+                         'None defaults to the background from after all '
+                         'hierarchical removals performed. [default=None]')
```

Comment:
You can't "choose" None on the command line. Just put `If not provided, the remaining background after all hierarchical removals will be plotted.` After that, let me know when you're ready for this to be reviewed again.